aces/Loris
# Add Alias in parameter type table and map BIDS Alias to MINC headers present in parameter_type

## Brief summary of changes

A new column is added to the parameter_type to add an Alias to the parameters. For imaging related fields that comes from the MINC header of images, this Alias will be used to store the BIDS term mapping for a given parameter. This will be needed for the dcm2bids LORIS insertion pipeline in order to map the MINC terms with the BIDS terms.

It should be noted that currently, some fields are present multiple times in the parameter_type table with different parameter name (this can be observed in the raisin bread dataset). An example would be the echo time. There are 3 different parameter types available for the echo time: `echo_time` (ID 130), `acquisition:echo_time` (ID 113) and `dicom_0x0018:el_0x0081` (ID 209). For the BIDS terms, only the `echo_time` has the BIDS term Alias `EchoTime` to avoid mapping `EchoTime` randomly to 3 different parameter type ID.

We might want to think whether we should not clean those duplicata at some point for the MINC files. To be discussed at the imaging call on Thursday July 8th.

**All descriptions added to the parameter fields in this PR came from [InterLex](https://scicrunch.org/scicrunch/interlex/dashboard).**

#### Testing instructions 

1. Run the patch and check that the Alias and Description have been updated for the fields present in this PR
2. Test that 0000-00-000-schema.sql is running properly
3. Reload the raisin bread dataset and check that the Alias and Description have been updated for the fields present in this PR. 

#### Link(s) to related issue(s)

This is not linked to a particular issue but was discussed during a roadmap meeting. This feature will become handy for the dcm2bids LORIS insertion script.
